addons/website/views/snippets/s_banner_product.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_banner_product" name="Collection Banner">
+    <section
+        class="s_banner_product o_cc o_cc5 pt24 pb24 parallax"
+        data-scroll-background-ratio="0.2"
+        data-parallax-type="zoom_out"
+        >
+        <span class="s_parallax_bg oe_img_bg o_bg_img_center o_bg_img_origin_border_box" style="background-image: url('/web/image/website.shop_category_1_16x9'); top: -20px; bottom: -20px; transform: scale(1.097);"/>
+        <div class="o_we_bg_filter bg-black-15"/>
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="15">
+                <div
+                    class="o_grid_item g-height-4 g-col-lg-7 col-lg-7"
+                    style="z-index: 1; grid-area: 12 / 1 / 16 / 8;"
+                    >
+                    <h2>Explore our new collection</h2>
+                    <p>Simple forms. Natural textures. Timeless design. Explore our new furniture collection—crafted to bring calm and clarity to your space.</p>
+                </div>
+                <div
+                    class="o_grid_item g-height-4 g-col-lg-4 col-lg-4"
+                    style="z-index: 2; grid-area: 12 / 9 / 16 / 13; text-align: right;"
+                    >
+                    <a href="" title="" role="button" class="btn btn-lg btn-secondary">
```

Comment:
Changing to # href, no title and no button role (note that marking a link as a button is not only useless, it is a mistake)